src/metrics.go

```diff
@@ -232,6 +232,32 @@ func getDbAndKeys(keysFlag sdkArgs.JSON) map[string][]string {
 	return databaseKeys
 }
 
+// getRenamedCommands returns a map of command and renamed command pair
+// Example: '{"CONFIG":"ZmtlbmZ3ZWZl-CONFIG"}'
+func getRenamedCommands(renamedCommandsFlag sdkArgs.JSON) map[string]string {
+	renamedCommands := make(map[string]string)
+
+	convertF := func(stringInterface interface{}) string {
```

Comment:
I am aware this was done to keep a similar format as for other args, but I'm not a big fan of this `convertF`. For example, if the user writes a number as the value for an override, a warning will be logged but an empty string will be still registered as the override, breaking the integration later.

I think it's better if `convertF` can return `(string, error)`. This way, in the loop below, we can skip adding the override if an error is returned (e.g. value is not a string).

Reply:
makes sense, added the error check